tests/__tests__/parser/validate.test.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+require('babel-polyfill');
+const validateAddons = require('../../../lib/utils/validate-addons');
+
+describe('Addon Validation', () => {
+	it('Should notify if the package isn\'t prefixed', async () => {
+		try {
+			await validateAddons(['webpack']);
+		} catch (object) {
+			expect(object.toString()).toEqual('TypeError: Package isn\'t a valid name\n' +
```

Comment:
It's a thing I implemented to make the test run, ,see prev comment. This should be refactored, but was a nice way to get the error message that didn't pass using `toThrow`.